packages/forms-web-app/src/lib/determine-user.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {string} url
+ * @returns {import("../../../appeals-service-api/src/db/seed/data-static").AppealToUserRoles}
+ */
+const determineUser = (url) => {
+	/** @type {import("../../../appeals-service-api/src/db/seed/data-static").AppealToUserRoles} */
+	let userType = 'interestedParty';
+
+	if (url.includes('/manage-appeals/')) {
+		userType = 'agent';
+	} else if (url.includes('/rule-6-appeals/')) {
+		userType = 'rule6Party';
+	} else if (url.includes('/appeals/')) {
+		userType = 'appellant';
+	}
+
+	return userType;
```

Comment:
Do we need `userType` variable? Might be clearer to just return in each `if`, with a default return at the end?